build-system/babel-config/helpers.js

```diff
@@ -19,6 +19,21 @@ const argv = require('minimist')(process.argv.slice(2));
 const experimentsConfig = require('../global-configs/experiments-config.json');
 const experimentsConstantBackup = require('../global-configs/experiments-const.json');
 
+function getExperimentConstant() {
+  const flag = argv['define_experiment_constant'];
```

Comment:
nit: prefer dot access (`argv.define_experiment_constant`) for easy maintainability of flags.

Reply:
When using dot access, the linter complains about the property being snake case